### ISSUE TITLE
Failed to start tor.service: Unit tor.service not found.

### ISSUE THREAD

**Issue by rs9498** [Und3rf10w/kali-anonsurf#53]
i am not be able to use anonsurf it is installing well but after installing i am not be able to use it 

[ i ] Starting anonymous mode:

 * Tor is not running!  starting it  for you

Failed to start tor.service: Unit tor.service not found.
 * Saved iptables rules

 * Modified resolv.conf to use Tor and Private Internet Access DNS
iptables v1.6.1: owner: Bad value for "--uid-owner" option: "debian-tor"
Try `iptables -h' or 'iptables --help' for more information.
iptables v1.6.1: owner: Bad value for "--uid-owner" option: "debian-tor"
Try `iptables -h' or 'iptables --help' for more information.
iptables v1.6.1: owner: Bad value for "--uid-owner" option: "debian-tor"
Try `iptables -h' or 'iptables --help' for more information.
 * All traffic was redirected through Tor

[ i ] You are under AnonSurf tunnel

**Comment by pranid** [comment]
I reinstalled tor via `apt-get install -f tor`, and it worded for me

**Comment by weiyushit** [comment]
@pranid Hi there, after trying the command recommended, I received the following errors, what do you think? `root@kali:/home/test# apt-get install -f tor
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  libzstd1 tor-geoipdb torsocks
Suggested packages:
  mixmaster torbrowser-launcher tor-arm apparmor-utils obfs4proxy
The following NEW packages will be installed:
  libzstd1 tor tor-geoipdb torsocks
0 upgraded, 4 newly installed, 0 to remove and 420 not upgraded.
Need to get 3,084 kB/3,157 kB of archives.
After this operation, 12.4 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Err:1 http://http.kali.org/kali kali-rolling/main amd64 libzstd1 amd64 1.3.3+dfsg-2
  404  Not Found [IP: 192.99.200.113 80]
Err:2 http://http.kali.org/kali kali-rolling/main amd64 tor amd64 0.3.2.10-1
  404  Not Found [IP: 192.99.200.113 80]
Err:3 http://http.kali.org/kali kali-rolling/main amd64 tor-geoipdb all 0.3.2.10-1
  404  Not Found [IP: 192.99.200.113 80]
E: Failed to fetch http://http.kali.org/kali/pool/main/libz/libzstd/libzstd1_1.3.3+dfsg-2_amd64.deb  404  Not Found [IP: 192.99.200.113 80]
E: Failed to fetch http://http.kali.org/kali/pool/main/t/tor/tor_0.3.2.10-1_amd64.deb  404  Not Found [IP: 192.99.200.113 80]
E: Failed to fetch http://http.kali.org/kali/pool/main/t/tor/tor-geoipdb_0.3.2.10-1_all.deb  404  Not Found [IP: 192.99.200.113 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
`

**Comment by MANOJJANGID67** [comment]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]                                                                     
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]                                                                     
88% [4 Packages 15.0 MB/17.0 MB 89%]                                            Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:4 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/main i386 Packages [17.0 MB]
Get:5 http://ftp.yzu.edu.tw/Linux/kali kali-rolling/non-free i386 Packages [164 kB]
Reading package lists... Done                                                  
E: The repository 'http://download.rethinkdb.com/apt kali-rolling Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by LaurenceJJones** [comment]
I added the default repo list on kali website here a snippet:

Regular repositories

On a standard, clean install of Kali Linux, you should have the following entry present in /etc/apt/sources.list:

deb http://http.kali.org/kali kali-rolling main non-free contrib

You can find a list of official Kali Linux mirrors here.
Source repositories

In case you require source packages, you might also want to add the following repositories as well:
deb-src http://http.kali.org/kali kali-rolling main non-free contrib

TLDR: 
1) nano  /etc/apt/sources.list
2) add new line: "deb http://http.kali.org/kali kali-rolling main non-free contrib"
3) add another line: "deb-src http://http.kali.org/kali kali-rolling main non-free contrib"
4) Ctrl X and Y to save (make sure you didnt copy "" at end)
5) apt-get update
6) apt-get install tor
